kibana_k8s_proxy/main.tf

```diff
@@ -0,0 +1,183 @@
+provider "helm" {
```

Comment:
I don't think you need the helm provider in this module

Reply:
Idd, leftover 😓